core/dbt/parser/schema_yaml_readers.py

```diff
@@ -125,6 +144,102 @@ def __init__(self, schema_parser: SchemaParser, yaml: YamlBlock):
         self.schema_parser = schema_parser
         self.yaml = yaml
 
+    def _get_input_measure(
```

Comment:
These helper functions look good enough for a beta, but I think we should look at ways to make the code less repetitive and mechanical.

More specifically, if we are going to continue along these lines, it would be better to have a way to declare how the unparsed objects map to the parsed objects, rather than performing the conversion manually for these and every new object.